azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -114,22 +129,47 @@ func resourceArmNetAppVolume() *schema.Resource {
 								ValidateFunc: validate.CIDR,
 							},
 						},
+
+						"protocols_enabled": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Computed: true,
+							MaxItems: 2,
```

Comment:
The docs say only one value is supported at this time. Is that right? If so, we should match that here? From local testing, it seemed like the API accepted all three values being passed in at the same time.

Reply:
Hi @mbfrahry, to your first question, this will be supported in the future and @katbyte asked me to use TypeSet and I tried to reduce the items to 1 but TypeSet does not supports it, only TypeList.
Regarding the second part, as of today it is not supported to have more than one at this point, we have on the road map to support two, CIFS+NFS3 and CIFS+NFS4 but never the three of them or NFS3+NFS4 combination. Have you checked if the export policy got created? It will not throw an error, it just don't get the export policy created.

Reply:
Ah, that makes sense. Thanks for clarifying. I just saw the values were put into the template but I didn't deploy the template.